recipes/tassel/3.0.137/build.sh

```diff
@@ -0,0 +1,31 @@
+BINARY_HOME=$PREFIX/bin
+TASSEL_HOME=$PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM
+
+# Download TASSEL4 from its public repo
+git clone git://git.code.sf.net/p/tassel/tassel3-standalone tassel3-standalone
+cd tassel3-standalone
+git checkout V$PKG_VERSION
```

Comment:
Why isn't the source given in `meta.yaml` like so:
```yaml
source:
  git_url: https://git.code.sf.net/p/tassel/tassel3-standalone.git
  git_rev: V{{ version }}
```

Reply:
I'm not too familiar with Git on SourceForge, but is it possible (and reliable) to get source tarballs from there?

Reply:
Changed to download snapshot from SourceForge in the latest commit.